### DISCUSSION TITLE
How wide is the sidewalk stub here?

### DISCUSSION THREAD

**Discussion by ivanbranco** [Helium314/SCEE#630] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/fc6e4e92-ad19-4705-ae0b-482152b5069b)

Sometimes SCEE asks me "How wide is the foot path here?" for sidewalks mapped like this and I always ignore it because I'm not sure how to reply. Should I use the crossing markings width? Should I ignore the task? Should I change the footway from footway=sidewalk to footway=link?

Any ideas?

**Comment by mnalis** [comment, marked as answer]
I'd probably just ignore it, or measure the crossing markings (i.e. the same width as that leftmost `footway=crossing` horizontal way) if I was bored and had to wait for someone at that place, but as `footway=sidewalk` is dubious (if not flat our wrong) in that case, changing it to `footway=link` might perhaps be good idea instead. (or it might be bad idea -- see below!)

But [community forum tagging category](https://community.openstreetmap.org/c/general/tagging/70) might be better place to discuss it (and with wider community too); as determining how to tag that situation is not specific to SCEE...

**Reply by ivanbranco** [reply]
Done here: https://community.openstreetmap.org/t/how-to-tag-sidewalk-stubs/119025

**Comment by HolgerJeromin** [comment]
IMO it would be good if SC(EE?) could retag the way to `=link`.

**Reply by rusty-snake** [reply]
I relative new to OSM (three month) but saw newbie users doing funny thinks with such link/routing path/footway. A "Uhh ... > Does not exists, mark as `link`" for `highway ~ path|footway and "length" < 10meter` in SC(EE?) could make sense.
On the otherhand, people can survey the roof-color of houses that where never constructed. But for the expert users, it makes sense (yes, you can edit tags, but the CS comment will be much better).
